CMSgov/onemac
# Oy2 11589 - app k form in package view

Story: https://qmacbis.atlassian.net/browse/OY2-11589
Endpoint: See github-actions bot comment

### Details

Allow submission of 1915(c) App K Waiver Amendments 

### Changes

- Added waiver app k form
- Updated email format to include possible title field (as exists on app k form)


### Test Plan

1. Login as state submitter and navigate to package view
2. New Submission -> Waiver Action 
3. Verify App K card title and subtext
4. Click App K card and verify you are directed to an empty app k form
5. Verify Title and fields per ACs
6. Fill out all fileds other than Title and verify title acts as a required filed not allowing submit
7. Fill out Title and verify submit is now active
8. Submit AppK form
9. Verify you are redirected to the package list waiver tab with the green confirmation banner and that the App K which was submitted appears as a top level item in the package list.
10. Verify email is sent and the title is part of the details in the email.
